lib/simbricks/nicbm/nicbm.cc

```diff
@@ -559,10 +577,10 @@ Runner::Runner(Device &dev) : main_time_(0), dev_(dev), events_(EventCmp()) {
 
 int Runner::ParseArgs(int argc, char *argv[]) {
   if (argc < 4 || argc > 10) {
```

Comment:
```suggestion
  if (argc < 4 || argc > 11) {
```
Otherwise we will always just print the usage and exit with an error when specifying the log file path :slightly_smiling_face: